python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -557,14 +557,32 @@ def test_delete_date_range(arctic_library):
 
 
 @MONGO_TESTS_MARK
-def test_mongo_repr(mongo_test_uri):
-    max_pool_size = 10
-    min_pool_size = 100
-    selection_timeout_ms = 1000
-    uri = f"{mongo_test_uri}/?maxPoolSize={max_pool_size}&minPoolSize={min_pool_size}&serverSelectionTimeoutMS={selection_timeout_ms}"
+def test_mongo_repr_and_uri(mongo_test_uri):
+    uri = f"{mongo_test_uri}/?maxPoolSize=10&minPoolSize=100&serverSelectionTimeoutMS=1000"
     ac = Arctic(uri)
     assert repr(ac) == f"Arctic(config=mongodb(endpoint={mongo_test_uri[len('mongodb://'):]}))"
 
+    # With pymongo, exception thrown in the uri_parser;
+    with pytest.raises(UserInputException):
```

Comment:
It is for testing whether the extraction of endpoint is valid. 
With `pymongo`, `uri_parser` is used for the extraction.
Without `pymongo`, `re` is used. It is not really necessary but it will be easier to debug if something gets messed up.